[DOC-748] Support bundle phase 4 (#27672)
yugabyte/yugabyte-db: docs/content/v2025.1/yugabyte-platform/troubleshoot/universe-issues.md

Logs generated by YBA.

#### System logs

> Collects the `/var/log/messages` log files from all database nodes that fall within the support bundle's start and end date. This component is not applicable for kubernetes universes.

Reviewer: ```suggestion
Collects the `/var/log/messages` log files from all database nodes that fall within the support bundle's start and end date. This component is not applicable for Kubernetes universes.
```

The Prometheus metrics are stored in JSON files in the `support_bundle/YBA/promdump` directory.

> You can also create "Custom Queries" by providing PromQL expressions; their results will be stored in the support bundle under the corresponding folder name.

Reviewer: ```suggestion
You can also create "Custom Queries" by providing PromQL expressions; their results are stored in the support bundle under the corresponding folder name.
```
